### PR TITLE
Preview ITrace: Fixes Culture Info

### PR DESCRIPTION
**Pull request by bchong95**
# Preview ITrace: Fixes Culture Info

## Description

Fixes the culture info of ITrace text strings.

Resolves: https://github.com/Azure/azure-cosmos-dotnet-v3/issues/2223

### CODE REVIEW

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml [hunk 1]
```diff
@@ -197,15 +197,15 @@
...
+            ┌────────────────┬──────────────┬──────────────────────────────────────────────────┐
+            │Start Time (utc)│End Time (utc)│Endpoint                                          │
+            ├────────────────┼──────────────┼──────────────────────────────────────────────────┤
+            │    12:00:00:000│  11:59:59:999│                              http://localhost.com│
+            │    12:00:00:000│  11:59:59:999│                              http://localhost.com│
+            └────────────────┴──────────────┴──────────────────────────────────────────────────┘
             Store Response Statistics
-              Start Time: 0001-01-01T00:00:00.0000000
-              End Time: 9999-12-31T23:59:59.9999999
+              Start Time: 12:00:00:000
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml:207]: The date is critical for being able to correlate to the backend logs. Why is this being removed?

**Review by sboshra — APPROVED** [review]
:shipit:

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml (hunk 1 above)
Comment (ealsur) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml:207]: +1. We have history of CRIs on AKS where the pods go to sleep and awake again days later and these dates were critical to see that.

Comment (bchong95) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml:207]: The activity id is included. That can be used to get the exact date. Please provide an ICM where we needed the full date. This is also only on the text path and not json path.

Comment (ealsur) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml:207]: I think ActivityId is tracked for failures, but if the request was a success, it is not possible to find related time information (for a particular successful request).

How does ActivityId help to track the time difference between the request being initiated in the client machine and the response if the AKS pod is suspended and resumed with days in between (it sounds crazy but it happened and the date in this data points was what showed us the behavior).

Comment (j82w) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml:207]: @ealsur gave you an example of why it is necessary.  Why isn't this getting fixed for the JSON output? Is it already fixed for the JSON output?

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml:207]: Full UTC/GMT times are essential. Most of the times CX given impact times and what diagnostics says also differ. 
Also going forward lets please connect offline before any data removal to ensure we are all on common understanding.

/cc: @sboshra

Comment (bchong95) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/BaselineTest/TestBaseline/TraceWriterBaselineTests.TraceData.xml:207]: As stated this PR does not impact the JSON output. So any concern about the datetime for investigations is not relevant. You are also free to expand out the DateTime in a seperate PR. This PR mainly stabilizes the times for the baselines.

**Pull request merged**